pages/instructorScrapPaper/instructorScrapPaper.ejs

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <%- include('../partials/head'); %>
+	<title>Make Scrap Paper</title>
```

Comment:
I took a second look at the title stuff, we should be able to rely on the default title used on all pages - see https://github.com/PrairieLearn/PrairieLearn/blob/c4a120a217800c8dc4978b993cbc3ab2f1bffb10/pages/partials/head.ejs#L4

Reply:
Done in part 2.